src/containers/SettingList/Providers/Providers.test.tsx

```
@@ -70,7 +72,7 @@ describe('<Providers />', () => {
 });
 
 const maytapiProvider = (error: boolean = false) => {
-  let MOCKS: any = [...mocks, ...getMaytapiProviderMock, , createMaytapiCredentialsMock(error)];
+  const MOCKS: any = [...mocks, ...getMaytapiProviderMock, , createMaytapiCredentialsMock(error)];
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)